scripts/generate-docker-compose-credentials.sh

```diff
@@ -114,5 +114,6 @@ UI_ADMIN_USERNAME=admin
 UI_ADMIN_PASSWORD=$(gen_random_string)
 UI_DJANGO_SECRET_KEY=$(gen_random_string)
 UI_DB_PASSWORD=$(gen_random_string)
+VERSION=$( cat "VERSION")
```

Comment:
This value does not refer to UI, could you move it to another place in the file?